source/_guides/6-php-client-intro.md

```diff
@@ -0,0 +1,207 @@
+# PHP client library intro
+
+
+### CMS Clients
+
+Note: If you just want to use rokka in a CMS like Drupal, Wordpress or Kirby, we recommend using the respective plugins (add links). If you want to know, how the PHP library works and what it provides, this intro may be a good start nevertheless.
+
+## Preparation
+
+If you already know the basics of rokka and have uploaded an image and created a stack, you can directly skip to the "The PHP library" chapter below.
+
+### Get a rokka account
+
+Before we start with the actual PHP stuff, we do some preparation. First, you of course need a rokka account. Head over to [the signup page](https://rokka.io/dashboard/#/signup) and get one. It's free for 90 days and afterwards still free for up to 5 GB of traffic and 2 GB of storage.
+
+After you signed up, you'll get an email with your api key, don't loose that.
+
+### Install the rokka CLI
+
+For some basic setup, we'll use the rokka CLI, go and [get it](https://github.com/rokka-io/rokka-go) and follow the instructions to install it.
+
+Note: Alternatively you can also just login at [rokka.io/dashboard](https://rokka.io/dashboard) and do there what we will do here with the CLI.
+
+After you installed the rokka CLI, store your rokka api key in a config file.
+
+```
+rokka login --apiKey $YOUR_API_KEY
+```
+
+### Upload an image
+
+Now you can upload your first image to rokka. We recommend to upload a high resolution image in a lossless image format like PNG. But there's no use to convert eg. a JPEG to PNG, if the JPEG version is the only version you have.
+
+```
+rokka sourceimages create $YOUR_ORG rokka-pic.png
+``` 
+
+This will return some metadata information about the upload, eg:
+
+```
+Hash:         dba8937b9b057e419cf96c0696be8db9ed481218 (dba893)
+Name:         rokka-pic.png
+Details:      image/png, 1260x840, 894914Bytes
+Created at:   Wed, February 21 2018, 13:18:15Z
+Binary hash:  069a28acf55626285e6ade09064a661e6ce893f9
+```
+
+The important part is the hash (or the short hash in the parenthesis). The hash is an unique value for each image. If you upload the same image again, it will return you the same hash. If you upload an image where just one byte changed, it will return you a totally different hash.
```

Comment:
typo: `an unique` -> `a unique`

Reply:
> If you upload an image where just one byte changed
maybe replace the `where` with `with`, unsure about this one though.

Reply:
makes sense